skbio/alignment/_path.py

```diff
@@ -146,13 +146,20 @@ def to_indices(self, gap=-1):
 
         Parameters
         ----------
-        gap : -1, "del", or "mask", optional
-            If -1, replace gaps with this value. If "del", delete columns that have any
-            gap. If "mask", mask gaps. Default is -1.
+        gap : int, float, "del", or "mask", optional
```

Comment:
I think an index cannot be a float. It is possible to cast floats into integers, but it shouldn't be a normal use case.